bin/read-all-feeds

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env bash
-dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+
+# This do not work when running through  docker compose exec.
+# It defaults to /home/deploy and not the working dir.
+#
+#dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
```

Comment:
How and where does this not evaluate to `/app/bin` when run with `docker compose exec`?

I assume it's run as `docker compose --file docker-compose.server.yml exec phpfpm bin/read-all-feeds` and in this case the working directory must be `/app` (otherwise `bin/read-all-feeds` does not exist).

Reply:
See the servers cron job where it executes from.

It returns no-such-file `/home/deploy/console` @rimi-itk so the old way don't default to /app/bin as not bash context have been loaded...